src/test/scala/mesosphere/marathon/integration/facades/MesosFacade.scala

```diff
@@ -83,7 +83,7 @@ object MesosFacade {
   case class ITFrameworks(frameworks: Seq[ITFramework])
 }
 
-class MesosFacade(url: String, implicit val waitTime: FiniteDuration = 30.seconds)(implicit val system: ActorSystem, materializer: Materializer)
+class MesosFacade(val url: String, implicit val waitTime: FiniteDuration = 30.seconds)(implicit val system: ActorSystem, materializer: Materializer)
```

Comment:
Scala allows the mix of implicits like this? (even if it does... probably shouldn't :) :) )

Reply:
Now that I'm looking at it - it does look strange :) But my only change was to make `url` public.